README.md

```diff
@@ -0,0 +1,171 @@
+# Budget_app
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Project Live Presentation](#loom-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MP Budget App] <a name="about-project"></a>
+
+**[MP Budget App]** ia a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/en/documentation/">Ruby</a>
+- <a href="https://rubyonrails.org/">Rails</a>
+- <a href="https://www.postgresql.org/">PostgreSQL</a>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add budget category]**
+- **[Add Purchases]**
+- **[User Authentication]**
+- **[View category and purchases]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://mp-budget.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presentation -->
+## Project Presentation <a name="loom-demo"></a>
+
+- [Loom Walkthrough Link](https://loom.com/share/a5811b03ae1e4f809c29d794bcafed01)
```

Comment:
- [ ] [REQUIRED] Kindly ensure that your presentation video contains a highlight of a piece of code that you are proud of making as stated by the project requirements.

Reply:
Thank you @Whoistolu , i will create a new presentation and request another review

Reply:
- [ ] This is well resolved @mrprotocoll  🥇